src/popup/components/SelectAsset.vue

```diff
@@ -1,11 +1,11 @@
 <template>
   <BtnPlain
     class="select-asset"
-    :class="{ focused }"
+    :class="{ focused, enabled: !disabled }"
```

Comment:
Maybe inverting the logic would be better to do on the CSS level. So here we'll have only `{ focused, disabled }` and in the css:
```
:not(.disabled) {
```